stable/newrelic-infrastructure/templates/daemonset.yaml

```diff
@@ -39,13 +39,26 @@ spec:
                   key: license
             - name: "CLUSTER_NAME"
               value: "{{ .Values.cluster }}"
+           {{- if semverCompare "<1.7-6" .Capabilities.KubeVersion.GitVersion }}
```

Comment:
Shouldn't this be `<1.7.6-0`? However, you should remove this because the chart will only install on 1.8+ anyways.

Reply:
I've removed the block altogether.